http/headers/Clear-Site-Data.json

```
@@ -95,11 +95,17 @@
               "edge": {
                 "version_added": "≤79"
               },
-              "firefox": {
-                "version_added": "63",
-                "version_removed": "94",
-                "impl_url": "https://bugzil.la/1838506"
-              },
+              "firefox": [
+                {
+                  "version_added": "preview",
+                  "impl_url": "https://bugzil.la/1942272"
```

I think the bug reference is not needed unless you provide a bug that targets shipping this to release.

```suggestion
                  "version_added": "preview"
```